website/docs/add-secure-apps/flows-stages/stages/identification/index.md

```diff
@@ -16,7 +16,11 @@ Select which fields the user can use to identify themselves. Multiple fields can
 
 ## Password stage
 
-To prompt users for their password on the same step as identifying themselves, a password stage can be selected here. If a password stage is selected in the Identification stage, the password stage should not be bound to the flow.
+To prompt users for their password on the same step as identifying themselves, a Password stage can be selected here. If a Password stage is selected in the Identification stage, the Password stage should not be bound to the flow.
+
+## Captcha stage
+
+To run a captcha in the background while the user is inputting their identification, a Captcha stage can be selected here. If a Captcha stage is selected in the Identification stage, the Captcha stage should not be bound to the flow.
```

Comment:
Suggestion (and maybe @tanberry can chime in): "entering" instead of "inputting".  "Inputting" is such an ugly phrase.  Either that or "... while the user inputs their identification, ..."  but that's still mechanizing the human, and I like my humans to stay human.